src/data.yaml

```diff
@@ -2976,3 +2976,33 @@ jobportals:
   - name: Stellenmarkt-Umweltschutz.de
     website: https://stellenmarkt-umweltschutz.de/
     country: Germany
+  - name: LinkedIn Green Jobs
+    website: https://www.linkedin.com/jobs/collections/green-jobs/
+    country: Worldwide
+  - name: Hyperion Search
+    website: https://hyperionsearch.co.uk/latest-jobs/
+    country: Worldwide
+  - name: SR2 - Socially Responsible Recruitment - B Corp
+    website: https://sr2rec.com/jobs/
+    country: UK
+  - name: SR2 - Socially Responsible Recruitment - B Corp
+    website: https://sr2rec.com/us/
+    country: USA
```

Comment:
duplicate entry

Reply:
This entry/URL was the USA specific job board - albeit from the same recruitment company as entry above which is the UK arm.

Happy to remove the USA one if you prefer